frontend/src/views/GShootItem.vue

```diff
@@ -45,27 +41,21 @@ export default {
     GPositionalDropzone,
   },
   provide () {
+    const terminalSplitpanes = this.terminalSplitpanes
     return {
-      terminalCoordinates: this.terminalCoordinates,
-      defaultTarget: this.defaultTarget,
-      splitpanesState: this.state,
-      moveTo: this.moveTo,
-      add: this.add,
-      setSelections: this.setSelections,
-      removeWithId: this.removeWithId,
-      leavePage: this.leavePage,
+      terminalSplitpanes,
+      ...terminalSplitpanes,
```

Comment:
was this done temporarily and in the future we only want to provide `terminalSplitpanes`

Reply:
Removed provide of `terminalSplitpanes` since it is not use at the moment.